helix-term/src/ui/mod.rs

```diff
@@ -82,21 +82,40 @@ pub fn file_picker(root: PathBuf) -> Picker<PathBuf> {
         Ok(entry) => {
             // filter dirs, but we might need special handling for symlinks!
             if !entry.file_type().map_or(false, |entry| entry.is_dir()) {
-                Some(entry.into_path())
+                let mut atime = None;
+                if let Ok(metadata) = entry.metadata() {
+                    if let Ok(atime_) = metadata.accessed() {
+                        atime = Some(atime_)
+                    }
```

Comment:
In what cases this won't appear? Should we default it to a value?

Reply:
@pickfire `atime` is either [only updated on actual file write or only after a particular threshold](https://unix.stackexchange.com/a/8842/246284). So simply accessing opening a file will not update it.

Reply:
Why not have a fallback with ctime and with 0 epoch time so we can remove the `Some` and make it mandatory?

Reply:
Good idea, this way it will show even more relevant results up top. Refactored, let me know if there is anything I can still improve.